scio-test/src/it/scala/com/spotify/scio/benchmarks/ScioBenchmark.scala

```diff
@@ -232,8 +232,11 @@ object DatastoreLogger {
 
   lazy val Storage: Datastore = DatastoreHelper.getDatastoreFromEnv
   val Kind = "Benchmarks"
-  val OrderByBuildNumQuery = s"SELECT * from ${Kind}_%s ORDER BY buildNum DESC LIMIT 2"
-  val WhereBuildNumQuery = s"SELECT * from ${Kind}_%s WHERE buildNum = %s"
+
+  def latestBuildNumQuery(benchmarkName: String) =
```

Comment:
can these be private? the stylecheck is failing since they're public with no return type :)